vendor/OWNERS

```diff
@@ -2,3 +2,5 @@
 
 approvers:
 - dep-approvers
+reviewers:
+- logicalhan
```

Comment:
I'm a big +1 on expanding the reviewer pool. 

That said, this file is generated, and we probably want to manage this as a group.

I'd suggest adding a `dep-reviewers` alias, and updating `update-vendor.sh` to add
```
reviewers:
- dep-reviewers
```